sanic/worker/manager.py

```diff
@@ -18,7 +17,7 @@
 
 
 class WorkerManager:
-    THRESHOLD = 50
+    THRESHOLD = 300  # == 30 seconds
```

Comment:
Thoughts on this interval?

Reply:
I think it's fine. 30 seconds seems reasonable.